lib/rickAndMortyApi.js

```diff
@@ -0,0 +1,28 @@
+const request = require('superagent');
+
+
+const getCharacter = async id => {
+  const { body } = await request.get(`https://rickandmortyapi.com/api/character/${id}`);
+  return {
+    id: body._id,
+    name: body.name,
+    status: body.status,
+    species: body.species
+  };
+};
+
+const getManyCharacters = async id => {
+  const { body } = await request.get(`https://rickandmortyapi.com/api/character/${[id, id, id}`);
```

Comment:
This won't quit work because you can't add all of the ids to the url at the same time. You have to map through the id array and create a unique url for each one.